app/controllers/api/stages_controller.rb

```diff
@@ -18,10 +16,19 @@ def clone
     end
   end
 
+  def duplicable
```

Comment:
Situation:

You add a new deploy group(s) for testing. You need to know which stage is a complete stage (ie: all the commands make sense and it deploys the project successfully). Then, when you add your new stage, you simply clone the stage marked duplicable and change the deploy groups.

I wanted to call it a "Gold/Master Stage" but gold stage is meaningless and master stage was confusing.